fastchat/conversation.py

```diff
@@ -124,6 +125,14 @@ def get_prompt(self) -> str:
                 else:
                     ret += role + "\n"
             return ret
+        elif self.sep_style == SeparatorStyle.BILLA:
+            ret = self.system + self.sep
+            for role, message in self.messages:
+                if message:
+                    ret += role + ": " + message + self.sep
+                else:
+                    ret += role + ": " # must be end with a space
```

Comment:
@Neutralzz why is this space required? I tried `SeparatorStyle .ADD_COLON_SINGLE ` and found it works well.

Reply:
It matches the instruction-tuning format. If without a space, the output would sometimes be an empty string.